src/lib/prover/prover.ml

```diff
@@ -61,7 +76,7 @@ module Worker_state = struct
                    Wrap.input {Wrap.Prover_state.proof} Wrap.main
                    (Wrap_input.of_tick_field hash)
 
-               let extend_blockchain (chain : Blockchain.t)
+               let extend_blockchain (chain : Blockchain.Stable.Latest.t)
```

Comment:
My feeling is that nothing should really talk about versions unless it's explicitly about versioning. We should probably chat about it OOB, but I think `Blockchain.t` is really what we want here.

Reply:
reverted to `Blockchain.t` where possible in `Prover` and `Verifier.Prod`

We need to use the `Stable` module in the cases that use `bin_io` functions, but only there.

Reply:
In this specific case, it seems like this isn't relevant. Can you change this?